types/ovenplayer/tsconfig.json

```diff
@@ -0,0 +1,26 @@
+{
+    "compilerOptions": {
+        "module": "commonjs",
+        "lib": [
+            "es6",
+            "dom"
+        ],
+        "target": "es5",
+        "noImplicitAny": true,
+        "noImplicitThis": true,
+        "esModuleInterop": true,
```

Comment:
```suggestion
```
you no longer needs this, your test file has proper import. Users can add this to their tsconfig (and by default TSC compiler add this in `--init` command):
https://github.com/DefinitelyTyped/DefinitelyTyped#esmoduleinteropallowsyntheticdefaultimports